pom.xml

```diff
@@ -57,6 +58,16 @@
 				<artifactId>poi-ooxml</artifactId>
 				<version>3.17</version>
 			</dependency>
+			<dependency>
+				<groupId>life.qbic</groupId>
+				<artifactId>core-utils-lib</artifactId>
+				<version>1.6.0</version>
+			</dependency>
+			<dependency>
+				<groupId>life.qbic</groupId>
+				<artifactId>data-model-lib</artifactId>
+				<version>1.9.4</version>
+			</dependency>
```

Comment:
The `core-utils-lib` already includes `data-model-lib` this might lead to conflicts. Is it possible to make the `data-model-lib` imported by `core-utils-lib` accessible?

Reply:
try, but it is still the responsibility of the project developer to include the dependency. This pom only pins the version.

This is why we use Maven. The dependency resolver will take care of the rest. If we have a transitive dependency (like data-model-lib in core-utils-lib is older than the required data-model-lib, than you need to enforce it (by setting the version in your project explicitly. This however might lead to breaking core-utils-lib. But then for me I raise the question, why we have a core-util-lib that uses an out-of-date model ;)